django/contrib/staticfiles/checks.py

```diff
@@ -12,3 +14,17 @@ def check_finders(app_configs=None, **kwargs):
         else:
             errors.extend(finder_errors)
     return errors
+
+
+def check_storages(app_configs=None, **kwargs):
```

Comment:
Added as a separate function, rather than in the `FileSystemFinder.check`: I assume this check is not specific to the `FileSystemFinder`, unlike the others.